lisa/tools/ip.py

```diff
@@ -230,3 +235,21 @@ def setup_tap(self, name: str, bridge: str) -> None:
 
         # start interface
         self.up(name)
+
+    def get_interface_list(self) -> list[str]:
+        raise NotImplementedError()
+
+
+class IpFreebsd(Ip):
```

Comment:
Because we don't provide the `ip` tool for Linux. It should be in ethtool, or directly execute the command as "ip -l"

Reply:
Oh, I see. I thought ip is deprecating.